src/contract/mod.rs

```diff
@@ -28,7 +28,7 @@ mod state;
 mod anchor;
 pub mod seal;
 pub mod assignments;
-mod operations;
+pub(crate) mod operations;
```

Comment:
we do not need this since we can Import `Transition` and `Extension` from `crate::`: they are public anyway